.npmignore

```diff
@@ -1 +1,15 @@
-demo/
+.github/
+docs/
+generate-component-docs/
+report/
+scripts/
+site/
+dist-storybook/
+icons/
```

Comment:
Are you sure this won't exclude our icon components?

Reply:
Yep, checked that.